apps/authenticator/lib/sessions/manager.ex

```diff
@@ -116,7 +116,7 @@ defmodule Authenticator.Sessions.Manager do
 
     [status: "active", created_after: create_after, expires_before: now]
     |> Session.query()
-    |> Repo.update_all(set: [status: "expired"])
+    |> Repo.update_all(set: [status: "expired", updated_at: NaiveDateTime.utc_now()])
```

Comment:
`update_all/2`: "Keep in mind this update_all will not update autogenerated fields like the updated_at columns."